.github/workflows/docs.yml

```diff
@@ -13,14 +13,30 @@ jobs:
     - name: apt-update
       run: sudo apt-get update -qq
     - name: apt-get doxygen
-      run: sudo apt-get install -y doxygen
-    - name: build doc
+      run: sudo apt-get install -y doxygen graphviz
```

Comment:
Where is graphviz being used?

Reply:
graphviz is the project that creates the DOT graphing engines which doxygen will call to produce code graphs. See [here](https://awschult002.github.io/Nuklear/structnk__chart.html) for an example. The use of DOT is default for doxygen, but can be disabled in the Doxyfile. I personally, always like to have the graphs.